transcom/mymove
# MAIN - B-20282 prime sim edit button style fix

## [B-20282](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20282)

INT PR: https://github.com/transcom/mymove/pull/13413

## Summary

Fixed the styling issue in Prime Sim for the Edit buttons next to upload documents. The issue was happening when going to a move details as a TOO and then switching to Prime sim.

Before: 
![image](https://github.com/user-attachments/assets/505b51be-6406-4f4d-9cbb-7179da9283c0)

Now:
![image](https://github.com/user-attachments/assets/ebfb3682-e401-4446-8244-fbb19c315084)


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test
1. Use 'HHGMoveInSITDeparted' to create a move.
2. Start from the Prime Simulator page for the move where the 'Edit' button is showing and where the issue is supposedly present.
3. Click change user role and select TOO.
4. Click and any move and go to move details.
5. Click change user role and select prime simulator.
6. Click on the move you were on in step 2.
7. The "Edit" button should have a white background.


## Screenshots
![image](https://github.com/user-attachments/assets/2bdc3764-4731-4df9-b2b6-48de54959e13)
